app/test_main.py

```diff
@@ -1,3 +1,56 @@
+import pytest
+from typing import Any
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_years",
+    [
+        pytest.param(0, 0, [0, 0], id="Mistake during convertation"),
+        pytest.param(14, 14, [0, 0], id="Mistake during convertation"),
+        pytest.param(15, 15, [1, 1], id="Mistake during convertation"),
+        pytest.param(23, 23, [1, 1], id="Mistake during convertation"),
+        pytest.param(24, 24, [2, 2], id="Mistake during convertation"),
+        pytest.param(27, 28, [2, 2], id="Mistake during convertation"),
+        pytest.param(28, 29, [3, 3], id="Mistake during convertation"),
+        pytest.param(100, 100, [21, 17], id="Mistake during convertation"),
+        pytest.param(-3, -10, [0, 0], id="If negative number passed: return 0")
+    ]
+)
+def test_cat_age_and_dog_age_converted_to_human_age_correctly(
```

Comment:
`test_get_human_age_works_correctly` or simply `test_get_human_age`  would be a more suitable name imo